using-timescaledb/compression.md

```diff
@@ -271,15 +271,16 @@ You can see the results of the compression of that given chunk by running the fo
 
 ``` sql
 SELECT *
-FROM timescaledb_information.compressed_chunk_stats;
+FROM chunk_compression_stats('conditions');
 ```
 
-| hypertable_name | chunk_name | compression_status | uncompressed_heap_bytes | uncompressed_index_bytes | uncompressed_toast_bytes | uncompressed_total_bytes | compressed_heap_bytes | compressed_index_bytes | compressed_toast_bytes | compressed_total_bytes |
-| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |  --- |
-| my_hypertable | _timescaledb_internal._hyper_1_1_chunk | Compressed | 8192 bytes | 16 kB | 8192 bytes |   | 32 kB  | 8192 bytes   | 16 kB  | 8192 bytes | 32 kB |
-| my_hypertable | _timescaledb_internal._hyper_1_20_chunk | Uncompressed | | | | | | | | |
+| chunk_schema | chunk_name | compression_status | before_compression_table_bytes | before_compression_index_bytes | before_compression_toast_bytes | before_compression_total_bytes | after_compression_table_bytes | after_compression_index_bytes | after_compression_toast_bytes | after_compression_total_bytes | node_name |
+| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
+| _timescaledb_internal | _hyper_1_1_chunk | Compressed | 8192 bytes | 16 kB | 8192 bytes |  32 kB  | 8192 bytes   | 16 kB  | 8192 bytes | 32 kB | |
+| _timescaledb_internal | _hyper_1_20_chunk | Uncompressed | | | | | | | | | |
```

Comment:
Given the width, perhaps these should be shown in `\x` mode?

Reply:
I think in this instance, printing in `\x` would be highlighting the wrong thing.  Really we're just demonstrating the command, the actual numbers in the result aren't interesting, so I feel like minimizing the vertical space and allowing the user to scroll is a better fit.